### DISCUSSION TITLE
[Feature] iPhone client HDR support

### DISCUSSION THREAD

**Discussion by Pacoakm** [immich-app/immich#5500] [Feature Request]
### The feature

It is clear to see the HDR effects of photos and videos in the ios original photos app (especially the sunlight). Would it be possible to implement the HDR effects in the photos and videos preview of immich ios client?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by benmccann** [comment]
Probably depends on https://github.com/flutter/flutter/issues/91241. See also https://github.com/immich-app/immich/issues/5120 which discusses switching to `media_kit`

**Comment by bo0tzz** [comment]
#8484
